airflow/www/views.py

```diff
@@ -5158,6 +5164,12 @@ def varimport(self):
             flash(f"{suc_count} variable(s) successfully updated.")
             if fail_count:
                 flash(f"{fail_count} variable(s) failed to be updated.", "error")
+            if skipped:
+                flash(
+                    f"The variables with these keys: {list(skipped)} were skipped "
```

Comment:
```suggestion
                    skipped_repr = ", ".join(repr(k) for k in sorted(skipped))
                    f"The variables with these keys: {skipped_repr} were skipped "
```

We should this in the web UI so some formatting would be a good idea